@cjbackhouse  Please ensure that the c7 build works.  There is a problem:
/home/garren/scratch/larsoft/v08_59_00/srcs/webevd/webevd/WebEVD/PNGArena.h:42:24: error: type
      'std::__1::array<unsigned char, 89478484>' does not provide a subscript operator
      return (*data[i])[(y*kArenaSize+x)*4+c];
             ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~
/home/garren/scratch/larsoft/v08_59_00/srcs/webevd/webevd/WebEVD/PNGArena.h:47:24: error: type
      'std::__1::array<unsigned char, 89478484>' does not provide a subscript operator
      return (*data[i])[(y*kArenaSize+x)*4+c];
             ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~

--- a/webevd/WebEVD/PNGArena.h
+++ b/webevd/WebEVD/PNGArena.h
@@ -3,6 +3,7 @@
 
 #include "webevd/WebEVD/Temporaries.h"
 
+#include <array>
 #include <memory>
 #include <string>
 #include <vector>

